### PR TITLE
Env files for saml11-xpro, nucleo-l432kc and esp8266-esp-12x

### PR DESCRIPTION
**Pull request by MarcelStenzel**
This adds the .env files for the named boards in the title.

### CODE REVIEW

File: dist/etc/conf/esp8266-esp-12x.env [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+HIL_UART_DEV?=
+HIL_SPI_DEV?=0
+HIL_SPI_HW_CS?=
+HIL_I2C_DEV?=0
+HIL_AN_DEV?=
+HIL_AN_CHAN?=
+HIL_DAC_DEV?=
+HIL_DAC_CHAN?=
+HIL_PWM_DEV?=0
```
Comment (MrKevinWeiss) [inline dist/etc/conf/esp8266-esp-12x.env:81]: oops

File: dist/etc/conf/nucleo-l432kc.env [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+HIL_DUT_GPIO2_GPIO?="PA3/CN4-10/A2"
+HIL_DUT_GPIO2_PORT?=0
+HIL_DUT_GPIO2_PIN?=3
+
+HIL_UART_DEV?=1
+HIL_SPI_DEV?=0
+HIL_SPI_HW_CS?=
+HIL_I2C_DEV?=0
+HIL_AN_DEV?=
```
Comment (MrKevinWeiss) [inline dist/etc/conf/nucleo-l432kc.env:77]: Is it possible to define the adc and channel of the dut connected to the philip dac (if riot doesn't support it in the periph conf then leave it blank).

**Review by MrKevinWeiss — CHANGES_REQUESTED** [review]
A few changes

File: dist/etc/conf/nucleo-l432kc.env (hunk 2 above)
Comment (MarcelStenzel) [reply dist/etc/conf/nucleo-l432kc.env:77]: I couldn’t find anything about in the periph conf

**Review by MrKevinWeiss — APPROVED** [review]
Clarified looks good. ACK

**Pull request merged**